apps/admin-gui/src/app/vos/pages/group-detail-page/group-members/group-members.component.ts

```diff
@@ -75,12 +75,16 @@ export class GroupMembersComponent implements OnInit {
   inviteAuth: boolean;
   hideColumns: String[] = [];
 
+  statuses = new FormControl();
+  statusList = ['VALID', 'INVALID', 'EXPIRED', 'DISABLED'];
+  selectedStatuses: string[] = ['VALID', 'INVALID', 'EXPIRED', 'DISABLED'];
```

Comment:
Please let the selectedStatuses be only Valid and invalid. Thats how it works in old gui, and thats how it worked before your change in new gui. It could have some reason.